File: problemas/1000/1047.cs
using System;

namespace uri1047
{
    internal static class Program
    {
        private static void Main()
        {
            string[] entrada = Console.ReadLine().Split(' ');
            int minutoInicial = 60 * int.Parse(entrada[0]) + int.Parse(entrada[1]);
            int minutoFinal = 60 * int.Parse(entrada[2]) + int.Parse(entrada[3]);

            int duracao = minutoFinal > minutoInicial
                ? minutoFinal - minutoInicial
                : 1440 - minutoInicial + minutoFinal;

            int horas = duracao / 60;
            int minutos = duracao % 60;

            Console.WriteLine($"O JOGO DUROU {horas} HORA(S) E {minutos} MINUTO(S)");
        }
    }
}

File: problemas/1000/1047.php
<?php

$entrada = explode(' ', readline());
$minutoInicial = 60 * $entrada[0] + $entrada[1];
$minutoFinal = 60 * $entrada[2] + $entrada[3];

$duracao = $minutoFinal > $minutoInicial
    ? $minutoFinal - $minutoInicial
    :  1440 - $minutoInicial + $minutoFinal;

$horas = intdiv($duracao, 60);
$minutos = $duracao % 60;

echo("O JOGO DUROU {$horas} HORA(S) E {$minutos} MINUTO(S)\n");